e2e-tests/global-setup.js

```diff
@@ -27,23 +27,25 @@ async function globalSetup() {
 const authenticateToBoundary = async () => {
   // Log in and save the authenticated state to reuse in tests
   const browser = await chromium.launch();
-  const page = await browser.newPage();
+  const browserContext = await browser.newContext();
```

Comment:
This is an unrelated change but I saw it's [recommended](https://playwright.dev/docs/api/class-browser#browser-new-page) to create your own [context](https://playwright.dev/docs/api/class-browser#browser-new-context) and close it directly. In this case, I don't think it matters that much but I saw we needed the context to grab the `storageState` on line 39 anyways so we might as well manually control it.